libc/include/sys/time.yaml

```
@@ -12,3 +13,20 @@ functions:
     arguments:
       - type: const char*
       - type: const struct timeval*
+
+  - name: setitimer
+    standards:
+      - POSIX
+    return_type: int
+    arguments:
+      - type: int
+      - type: const struct itimerval *
+      - type: struct itimerval *
```

These should be restrict
```suggestion
      - type: const struct itimerval *__restrict
      - type: struct itimerval *__restrict
```

---

Fixed, should be good to land?

---

Yes, you're good to land. Please wait for the presubmits to finish, though if the buildkite is taking a long time you can skip that one.

---

I don't have merge access, could you merge it for me? Thanks